libslirp will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libslirp it will fail in the following way:

```
2024/11/12 23:12:23 INFO running step "test/pkgconf"
2024/11/12 23:12:23 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:23 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:23 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:23 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:23 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + basename /home/build/melange-out/libslirp-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + dev_pkg=libslirp-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + apk info -L libslirp-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + grep -q ^Name: usr/lib/pkgconfig/slirp.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + basename usr/lib/pkgconfig/slirp.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + lib_name=slirp uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + echo usr/lib/pkgconfig/slirp.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + grep -q '^usr/lib/pkgconfig/slirp.pc$\|^usr/share/pkgconfig/slirp.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + pkgconf --exists slirp uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN + pkgconf --print-errors --exists slirp uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN Package glib-2.0 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN Perhaps you should add the directory containing `glib-2.0.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 WARN Package 'glib-2.0', required by 'slirp', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:23 INFO pod 654d1e44e74bbdf4dc0df78d54f7ca37cb33a41cc8bcd9693a2c7181c8d1ff46 terminated
2024/11/12 23:12:23 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:23 INFO   workspace dir: /home/user/tmp/melange-workspace-340961848
2024/11/12 23:12:23 INFO   guest dir: /home/user/tmp/melange-guest-3957811294
2024/11/12 23:12:23 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libslirp] Error 1
```

--- a/libslirp.yaml
+++ b/libslirp.yaml
@@ -1,7 +1,7 @@
 package:
   name: libslirp
   version: 4.8.0
-  epoch: 0
+  epoch: 1
   description: A general purpose TCP-IP emulator
   copyright:
     - license: BSD-3-Clause
@@ -35,8 +35,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - glib-dev
         - libslirp
     description: libslirp dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

